css-view-transitions-1/Overview.bs

```diff
@@ -1600,14 +1600,14 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 		1. Let |callbackPromise| be null.
 
+		1. If |transition|'s [=ViewTransition/phase=] is not "`done`", then set |transition|'s [=ViewTransition/phase=] to "`update-callback-called`".
```

Comment:
May as well move it further up to keep the code using `callbackPromise` together?

Reply:
Sure, done